tests/fxl-spread_auto/OPS/page_001.xhtml

```diff
@@ -0,0 +1,10 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>rendition:spread auto</title>
+    <meta name="viewport" content="width=900, height=600" />
+    <link rel="stylesheet" href="fixed.css" />
+  </head>
+  <body id="p1">
+  	<p>Page 1. This test passes if the content is displayed either by spine item or in synthetic spreads, depending on reading system orientation or preferences.</p>
```

Comment:
I think this implies "no matter what happens, this test passes," no?

My interpretation of the spec would be that this should only pass if the RS *ever* uses spreads when set to auto (and it's acceptable not to pass).

Reply:
My interpretation is that auto is basically "RS Choice" and RS choice is often dependent on orientation, screensize, and user settings (default or otherwise), so yeah, this test is very hard to define.

Reply:
Much better. Because of ambiguity in "either" (which could again be read as "implementing either mode is sufficient even if you never switch"), maybe rephrase like one of these?

... is displayed conditionally either by spine item or in synthetic spreads ...
or
... is displayed by spine item and in synthetic spreads ...